addons/base_address_extended/tests/test_street_fields.py

```diff
@@ -36,3 +36,26 @@ def test_partner_create(self):
             partner.street_number = number
             partner.street_name = name
             self.assertEqual(partner.street, street.strip(), 'Wrongly formatted street: expected %s, received %s' % (street, partner.street))
+
+    def test_city_id_onchange_and_children_sync(self):
```

Comment:
are we testing onchange here? I see the onchange setting `city`, `zip` and `state_id` which are not tested. `test_child_sync` maybe?

Reply:
Thanks for the review.

Indeed. I simplified the test and forgot to rename it. :)

I've updated the name. ;)